tests/ppx/react/expected/newtype.res.txt

```diff
@@ -0,0 +1,6 @@
+@obj external makeProps: (~a: a, ~b: 'b, ~key: string=?, unit) => {"a": a, "b": 'b} = ""
```

Comment:
Note this won't compile: type `a`.

Reply:
Yes, the transform is incorrect. Will revert.

Reply:
@glennsl reverted the PR for now, we'll need a new PR with a fix.

Reply:
Crap. Shouldn't have assumed the tests were run through the compiler. I'll fix. I suspect this will make it a much more invasive change though.

Reply:
Do we also have a concrete piece of UI code to verify the result?

Reply:
I can certainly make one. Is there anywhere I can put this, or do you just want something to test manually on the side?

Reply:
Just a gist is ok, we don't have "full stack integration tests", so manual it is.